django_zxcvbn_password_validator/zxcvbn_password_validator.py

```diff
@@ -56,7 +56,7 @@ def append_translated_feedback(old_feedbacks, feedback_type, new_feedbacks):
                     new_feedbacks = [new_feedbacks]
                 for new_feedback in new_feedbacks:
                     old_feedbacks.append(
-                        f"{feedback_type} : {translate_zxcvbn_text(new_feedback)}"
+                        f"{feedback_type} {translate_zxcvbn_text(new_feedback)}"
```

Comment:
```suggestion
                        f"{feedback_type}:{translate_zxcvbn_text(new_feedback)}"
```
Right ?

Reply:
Hmm, the best solution would be to have the colon inside the localized string. Will change that.